### DISCUSSION TITLE
Lock "Difficulty Adjust" settings regardless of rate change

### DISCUSSION THREAD

**Discussion by yeahirai** [ppy/osu#31227] [Ideas] [answered]
Would be nice if the changes made using the "difficulty adjust" mod would be the final values of the map instead of the original that then gets changed after rate changes mod are applied like DT or HT. 

Personally i use the rate changes a lot when practicing maps in very small changes like 1,15x then 1,20 etc. But everytime i do that i need to do the math in my mind to guess what value to set in the Approach Rate to achieve the AR that is in my goal (unless i keep going back and forth from the mod select and the song select to see what value was the result, which i think is very stupid). 

This seems like an unnecessary step to take given that if i'm changing the AR already why not change the final value instead of doing (AR*2) + (13) / (3) in my mind for every distinct rate change that i do.

**Comment by Joehuu** [comment, marked as answer]
This was already requested in https://github.com/ppy/osu/discussions/15249, please upvote that!
